litho-it/src/test/java/com/facebook/litho/widget/RecyclerBinderWrapContentTest.java

```diff
@@ -902,7 +907,8 @@ public void testOnDataRenderedWithNoChanges() {
     recyclerBinder.notifyChangeSetComplete(false, changeSetCompleteCallback);
 
     verify(changeSetCompleteCallback).onDataRendered(eq(true), anyLong());
-    verify(mRecyclerView, never()).postOnAnimation(recyclerBinder.mRemeasureRunnable);
+    verifyPostOnAnimationWasCalledAtleastNTimesWith(
```

Comment:
This checks for `>= 0` and we need exactly `0`

Reply:
Still valid comment

Reply:
Done